webapp/packages/plugin-connections/src/ConnectionForm/DriverProperties/ConnectionDriverPropertiesTabService.ts

```diff
@@ -80,6 +80,12 @@ export class ConnectionDriverPropertiesTabService extends Bootstrap {
     const config = contexts.getContext(connectionConfigContext);
 
     config.properties = { ...state.config.properties };
+
+    for (const key of Object.keys(config.properties)) {
+      const value = config.properties[key];
+      delete config.properties[key];
```

Comment:
Why do we need to delete the property first and trim the key here?

Reply:
maybe we need to check if property is custom or not and do not trim properties that we get from the server (keep as is)